test/quantization/core/test_quantized_module.py

```diff
@@ -98,6 +98,9 @@ def _test_linear_api_impl(self, batch_size, in_features, out_features, use_bias,
                                              axis=0, dtype=torch.qint8)
         else:
             W_q = torch.quantize_per_tensor(W, 0.1, 4, torch.qint8)
+            # ONEDNN only supports symmetric quantization of weight
+            if torch.backends.quantized.engine == 'onednn':
+                W_q = torch.quantize_per_tensor(W, 0.1, 0, torch.qint8)
```

Comment:
would L100 error out since it's not symmetric quantization?

Reply:
I think L100 is OK.

> maybe we can select scale/zero_point based on qengine instead of hardcode them here

Do you mean something like this?
```zp_weight = 0 if qengine_is_onednn() else torch.randint(1, 10, (1,)).item()```

Reply:
yes exactly